cc/toolchains/llvm_x86_64_windows/BUILD.bazel

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2024 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+load(":config.bzl", "config")
+
+filegroup(name = "empty")
+
+constraint_value(
+    name = "llvm_mingw_toolchain",
+    constraint_setting = "@rules_swiftnav//cc/constraints:toolchain",
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "wrappers",
+    srcs = glob([
+        "wrappers/**",
+    ]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "ar_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:ar",
+    ],
+)
+
+filegroup(
+    name = "as_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:as",
+    ],
+)
+
+filegroup(
+    name = "compiler_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:bin",
+        "@llvm_mingw_toolchain//:include",
+    ],
+)
+
+filegroup(
+    name = "dwp_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:dwp",
+    ],
+)
+
+filegroup(
+    name = "linker_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:ar",
+        "@llvm_mingw_toolchain//:clang",
+        "@llvm_mingw_toolchain//:gcc",
+        "@llvm_mingw_toolchain//:ld",
+        "@llvm_mingw_toolchain//:lib",
+    ],
+)
+
+filegroup(
+    name = "objcopy_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:objcopy",
+    ],
+)
+
+filegroup(
+    name = "strip_files",
+    srcs = [
+        ":wrappers",
+        "@llvm_mingw_toolchain//:strip",
+    ],
+)
+
+filegroup(
+    name = "all_files",
+    srcs = [
+        ":compiler_files",
+        ":linker_files",
+        "@llvm_mingw_toolchain//:bin",
+    ],
+)
+
+config(name = "config")
+
+cc_toolchain(
```

Comment:
Question: Comparing this with other toolchain files, I notice there is no call to `cc_toolchain_config`, why is that?

Reply:
There is, 
https://github.com/swift-nav/rules_swiftnav/blob/sokhealy/mingw_support/cc/toolchains/llvm_x86_64_windows/config.bzl#L130-L142
I based this toolchain off of the other cross-compile toolchain that we are using.